PWGDQ/DataModel/ReducedInfoTables.h

```diff
@@ -401,6 +401,32 @@ using Dilepton = Dileptons::iterator;
 using DileptonExtra = DileptonsExtra::iterator;
 using DimuonAll = DimuonsAll::iterator;
 
+// candidate information
+namespace reducedcandidate
```

Comment:
We should give a more descriptive name here. Something like "reducedtriplet" or "dileptonTrackCandidate"

Reply:
Done!